backoffice/src/Repositories/LetterRepository.cs

```diff
@@ -32,7 +33,18 @@ public IQueryable<Letter> GetLettersWithAssociations()
     {
         return GetLetters()
             .Include(l => l.Droit);
-        
+    }
+
+
+    public IQueryable<Letter> GetLettersForCurrentUser(Guid currentUserId)
```

Comment:
This should be called something more descriptive. 

Maybe something like 

GetQualityAssuredLettersToSend